NodeTypes/Field/Honeypot/Honeypot.fusion

```diff
@@ -17,12 +23,14 @@ prototype(Sitegeist.PaperTiger:Field.Honeypot) < prototype(Neos.Neos:ContentComp
             condition = true
             renderer = afx`
                 <Neos.Fusion.Form:FieldContainer
```

Comment:
Why not just use a `div` ?

Reply:
I think it it easier to understand for non fusion-form pros like you…

Reply:
Not much opinion here from my side so div is fine.

Reply:
Done in #24